src/components/Tiles/TileGroup.js

```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router-dom';
 
 import Tile from './Tile';
 
-const TileGroup = ({ title, titleUrl, description, tiles, compact, intl }) => {
+const TileGroup = ({ title, titleUrl, description, tiles, compact, intl, content_type }) => {
```

Comment:
where are we using content_type?

Reply:
once upon a time, we were, now, we're not. removed!